util/execdetails/execdetails.go

```diff
@@ -971,3 +1003,41 @@ func (e *RuntimeStatsWithCommit) formatBackoff(backoffTypes []fmt.Stringer) stri
 	sort.Strings(tpArray)
 	return fmt.Sprintf("%v", tpArray)
 }
+
+// FormatDurationForExplain uses to format duration, this function will prune precision before format duration.
+// Pruning precision is for human readability. The prune rule is:
+// 1. if the duration was less than 1us, return the original string.
+// 2. readable value >=10, keep 1 decimal, otherwise, keep 2 decimal. such as:
+//    9.412345ms  -> 9.41ms
+//    10.412345ms -> 10.4ms
+//    5.999s      -> 6s
+//    100.45µs    -> 100.5µs
+func FormatDurationForExplain(d time.Duration) string {
```

Comment:
Not sure, but this function may be useful in elsewhere I guess. For example, logs?

Reply:
How about rename to `FormatDurationWithPrune` or `FormatDurationForReadability`?

Reply:
How about simply `FormatDuration`?

Reply:
Great, done.